libs/elements.py

```diff
@@ -140,6 +140,14 @@ def add_sample(self, metricId, ts, value, metricType, sign=None,
                 elif 'ty' in t:
                     self.element.type = t['ty']
 
+                # check for application version
+                elif 'ver' in t:
```

Comment:
because the protocol is size sensitive, I would use `v` but other than that it looks good